scripts/clatd.systemd

```diff
@@ -12,6 +12,8 @@
 Description=464XLAT CLAT daemon
 Documentation=man:clatd(8)
 After=network-online.target
+Before=nss-lookup.target
+Wants=network-online.target nss-lookup.target
```

Comment:
According to the `systemd.special(7)` manual page:

> All services for which the availability of full host/network name resolution is essential should be ordered after this target, ***but not pull it in***. (emphasis mine)

Therefore, I think it is wrong to add it to `Wants`.